tests/integration_tests/test_collection_hybrid_search.py

```
@@ -0,0 +1,431 @@
+        if dimension is not None:
+            config = HNSWConfiguration(dimension=dimension, distance='l2')
+            collection = client.create_collection(
+                name=collection_name,
+                configuration=config,
+                embedding_function=None
+            )
+        else:
+            collection = client.create_collection(name=collection_name)
+        return collection, collection.dimension
+    
+    def _generate_query_vector(self, dimension: int, base_vector: List[float] = [1.0, 2.0, 3.0]) -> List[float]:
```

Ruff (0.14.8)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)